test/JDBC/expected/BABEL-4888-vu-verify.out

```diff
@@ -0,0 +1,145 @@
+
+
+INSERT INTO babel_4888_t1 VALUES (9999, 'random', NULL)
+GO
+~~ROW COUNT: 1~~
+
+
+
+-- table 2 Combination of id1 & id2 should be unique
+INSERT INTO babel_4888_t2 VALUES (-1, '-a', 0x0001), (1, '+a', 0x1001), (NULL, '', NULL), (0, NULL, NULL)
+GO
+~~ROW COUNT: 4~~
+
+
+INSERT INTO babel_4888_t2 VALUES (NULL, '', 0x9999)
+GO
+~~ERROR (Code: 2627)~~
+
+~~ERROR (Message: duplicate key value violates unique constraint "babel_4888_t2_id1_id2_key")~~
+
+
+INSERT INTO babel_4888_t2 VALUES (0, NULL, 0x9999)
+GO
+~~ERROR (Code: 2627)~~
+
+~~ERROR (Message: duplicate key value violates unique constraint "babel_4888_t2_id1_id2_key")~~
+
+
+INSERT INTO babel_4888_t2 VALUES (NULL, NULL, NULL)
+GO
+~~ROW COUNT: 1~~
+
+
+INSERT INTO babel_4888_t2 VALUES (NULL, NULL, 0x9999)
+GO
+~~ERROR (Code: 2627)~~
+
+~~ERROR (Message: duplicate key value violates unique constraint "babel_4888_t2_id1_id2_key")~~
+
+
+
+-- table 3 id1 & id2 should be individually unique
+INSERT INTO babel_4888_t3 VALUES (-1, '-a', 0x0001), (1, '+a', 0x1001), (NULL, '', NULL), (0, NULL, NULL)
+GO
+~~ERROR (Code: 515)~~
+
+~~ERROR (Message: null value in column "id1" of relation "babel_4888_t3" violates not-null constraint)~~
+
+
+INSERT INTO babel_4888_t3 VALUES (NULL, '', 0x9999)
+GO
+~~ERROR (Code: 515)~~
+
+~~ERROR (Message: null value in column "id1" of relation "babel_4888_t3" violates not-null constraint)~~
+
+
+INSERT INTO babel_4888_t3 VALUES (0, NULL, 0x9999)
+GO
+~~ROW COUNT: 1~~
+
+
+INSERT INTO babel_4888_t3 VALUES (NULL, NULL, NULL)
+GO
+~~ERROR (Code: 515)~~
+
+~~ERROR (Message: null value in column "id1" of relation "babel_4888_t3" violates not-null constraint)~~
+
+
+INSERT INTO babel_4888_t3 VALUES (NULL, NULL, 0x9999)
+GO
+~~ERROR (Code: 515)~~
+
+~~ERROR (Message: null value in column "id1" of relation "babel_4888_t3" violates not-null constraint)~~
+
+
+
+-- table 4 Combination of id1 & id2 should be unique
+INSERT INTO babel_4888_t4 VALUES (-1, '-a', 0x0001), (1, '+a', 0x1001), (NULL, '', NULL), (0, NULL, NULL)
+GO
+~~ERROR (Code: 515)~~
+
+~~ERROR (Message: null value in column "id1" of relation "babel_4888_t4" violates not-null constraint)~~
+
+
+INSERT INTO babel_4888_t4 VALUES (NULL, '', 0x9999)
+GO
+~~ERROR (Code: 515)~~
+
+~~ERROR (Message: null value in column "id1" of relation "babel_4888_t4" violates not-null constraint)~~
+
+
+INSERT INTO babel_4888_t4 VALUES (0, NULL, 0x9999)
+GO
+~~ERROR (Code: 515)~~
+
+~~ERROR (Message: null value in column "id2" of relation "babel_4888_t4" violates not-null constraint)~~
+
+
+INSERT INTO babel_4888_t4 VALUES (NULL, NULL, NULL)
+GO
+~~ERROR (Code: 515)~~
+
+~~ERROR (Message: null value in column "id1" of relation "babel_4888_t4" violates not-null constraint)~~
+
+
+INSERT INTO babel_4888_t4 VALUES (NULL, NULL, 0x9999)
+GO
+~~ERROR (Code: 515)~~
+
+~~ERROR (Message: null value in column "id1" of relation "babel_4888_t4" violates not-null constraint)~~
```

Comment:
we should also validate both asc and desc ordering, null first and last ordering

Reply:
Actually these combinations are already being tested. If you check the prepare file, table `babel_4888_t1` for example, has `id1` in `DESC` order and `id2` in `ASC` order.

Reply:
but how are we validating that NULLs are actually at right places and index have right ordering?

Reply:
We are verifying it with the index definition, check the SELECT command below.